networking/eth2/src/main/java/tech/pegasys/teku/networking/eth2/ActiveEth2P2PNetwork.java

```diff
@@ -179,11 +179,17 @@ public void onSyncStateChanged(final boolean isInSync, final boolean isOptimisti
     gossipForkManager.onOptimisticHeadChanged(isOptimistic);
   }
 
-  private boolean isCloseToInSync() {
+  @VisibleForTesting
+  boolean isCloseToInSync() {
+    final UInt64 currentEpoch = recentChainData.getCurrentEpoch().orElseThrow();
+    final int maxSeedLookahead = spec.getSpecConfig(currentEpoch).getMaxSeedLookahead();
```

Comment:
Nit: id probably call this something like maxLookaheadEpochs given the confusion (local var)